test/env.test.ts

```diff
@@ -50,15 +46,16 @@ describe('Env', () => {
             -invalid: TEST
       `);
 
-    expect(m.getErrors()).toEqual([
-      'Validation error occurred:',
-      '1) "deploy.processor[0].env.-invalid" is not allowed',
-      '2) "deploy.api.env.0invalid" is not allowed',
-    ]);
+    expect(error).toEqual(
+      new ManifestParsingError('Validation error occurred', [
+        '"deploy.processor.env.-invalid" is not allowed',
```

Comment:
Why is processor not an array now?

Reply:
Because validation is done before transformation